hack/versions-management/ansible_test_runner.sh

```
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+"$SCRIPT_DIR/github_auth.sh"
+
+python "$SCRIPT_DIR/ansible_test_runner.py" $([ "$DRY_RUN" = true ] && echo "--dry-run")
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)